### PR TITLE
std.os -> std.posix

### PR DESCRIPTION
**Pull request by linusg**
Best effort "fix stuff until it compiles", possible that some of this could be cleaned up further with the new upstream API layering.

### CODE REVIEW

File: src/main.zig [hunk 1]
```diff
@@ -15,7 +15,6 @@ const should_enable_signal_handling = !is_windows and builtin.os.tag != .wasi;
 
 const logger = std.log.scoped(.zigline);
 
-// std.os is a LIE!
```
Comment (linusg) [inline src/main.zig:18]: It no longer is :)

Comment (alimpfard) [reply src/main.zig:18]: o rly?

File: test.hist
Comment (alimpfard) [inline test.hist:1]: pls remove this?

Comment (linusg) [reply test.hist:1]: oops, done

File: src/main.zig (hunk 1 above)
Comment (linusg) [reply src/main.zig:18]: https://github.com/ziglang/zig/pull/19354

Comment (alimpfard) [reply src/main.zig:18]: I like where this is headed :eyes:

**Pull request merged**